tracks/python/exercises/say/mentoring.md

```diff
@@ -0,0 +1,127 @@
+# Mentoring
+
+## Problems and Challenges
+This problem requires the student to transform an integer (0 <= n < 1,000,000,000,000) into its full English equivalent, and raise an exception for values outside that range.
+Take note of the tests - the use of 'and' as a separator is not required, and adding it will cause the tests to fail, but it is suggested as an extension exercise. 
+
+## Reasonable Solution
+```
+digits = [""] + "one two three four five six seven eight nine".split()
+teens = "ten eleven twelve thirteen fourteen fifteen sixteen seventeen eighteen nineteen".split()
+tens = ["", ""] + "twenty thirty forty fifty sixty seventy eighty ninety".split()
+denominations = (9, "billion"), (6, "million"), (3, "thousand"), (2, "hundred")
+
+def say(number):
+    if 0 > number or number > 999999999999:
+        raise ValueError("input out of range")
+    if number < 20 and number >= 10:
+        return teens[number % 10]
+    if number < 100:
+        if number // 10 and number % 10:
```

Comment:
Is the first test here trying to do `number > 10`? Doing that directly would be more explicit and easier to understand.

Reply:
Not quite sure what you mean - numbers 11 to 19 inclusive are handled specially so line 17 is just to isolate those. Or are you referring to line 20? That checks whether both the 10s digit and the ones digit are non zero, since we then need to use a hyphen. It's also possible to check those values directly before using join, but that requires separate handling of the zero case.

Reply:
The first test on line 20. I believe `number // 10`  is testing `number >= 10` in a less direct and less obvious manner.

Reply:
This can be changed in a future PR